browser/src/control/Control.Tabs.js

```diff
@@ -180,6 +175,13 @@ L.Control.Tabs = L.Control.extend({
 							'Hide': this._menuItem['.uno:Hide'],
 							'movesheetleft': this._menuItem['movesheetleft'],
 							'movesheetright': this._menuItem['movesheetright'],
+							'Move': this._menuItem['.uno:Move'],
```

Comment:
no copy in multi tab?

Reply:
> no copy in multi tab?

There's only 1 dialog - the "Move or copy" dialog. When there's only a single tab, we title it "Copy" and disable the move button ... because there's nowhere to move it. You can see some more about these options ~line 100